clients/fortnite_tracker.py

```diff
@@ -61,14 +62,17 @@ async def _search_username(player_name):
     url = ACCOUNT_SEARCH_URL.format(username=player_name)
 
     async with aiohttp.ClientSession() as client:
-        async with client.get(url, headers=HEADERS) as r:
-            assert r.status == 200
-            r = await r.json()
+        async with client.get(url, headers=HEADERS, allow_redirects=False) as r:
+            assert r.status == 302
+            r = r.headers
 
     if not r:
         raise UserDoesNotExist("Username not found in FN Tracker")
```

Comment:
I can do this too later, but I think it'll help if we raised this too so it'll be logged if r.status returns 404 when the user can't be found

If a user does not exist right now I think assert will just raise and logs won't be clear

Reply:
What do you mean? 

So the search always return a 302 regardless of whether the name is the same or not.  It will only return 404 when the user can't be found.

Reply:
Don't quite understand. 

So the search endpoint that I'm using will return a 302 regardless of whether the name was different or not.  It will always return 404 if it doesnt exist.